packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -45,26 +45,27 @@ export function login({
   url?: string
   query?: { [s: string]: string }
 }) {
+  // if networkName is not specified we connect to default network from config
+  const network =
+    networkType === 'L1' ? getL1NetworkConfig() : getL2NetworkConfig()
+  const networkNameWithDefault = networkName ?? network.networkName
+
   function _startWebApp() {
-    startWebApp(url, query)
+    const sourceChain =
+      networkNameWithDefault === 'mainnet' ? 'ethereum' : networkNameWithDefault
```

Comment:
why do we need to do this? is it because `Ethereum` is `mainnet` from wagmi or what?

Reply:
query param expect `ethereum`, while metamask expect `mainnet`